app/_hub/kong-inc/confluent/overview/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+nav_title: Overview
+---
+
+With Kafka at its core, [Confluent](https://confluent.io) offers complete, fully managed, cloud-native data streaming that's available everywhere your data and applications reside.
+
+<!-- I modified this a bit to point to Apacha  Kafka installations -->
+Kong also provides a Kafka Log as well as Kafka Upstream plugin for publishing logs and messages to a Apache Kafka topic.
+
+* See [Kafka Log](/hub/kong-inc/kafka-log/)
+* See [Kafka Upstream](/hub/kong-inc/kafka-upstream/)
+
```

Comment:
Can this plugin only run on service-less routes? If so, we should add a note here:

```suggestion

{:.note}
> **Note**: This plugin must be enabled on a service-less route. 

```
And explain why you can't use a service.

Reply:
It was just an example, I rewrote it in a way that doesn't mention the service-less part. I can be enabled on a route with a service or on a service, one caveat is that when the plugin is enabled, the URL of the service is ignored because the plugin sends a kafka message to the configured server/cluster. Worth adding a note about that?

Reply:
>  one caveat is that when the plugin is enabled, the URL of the service is ignored because the plugin sends a kafka message to the configured server/cluster. Worth adding a note about that?

Definitely :)